db/migrate/20251028130433_add_omniauth_to_users.rb

```
@@ -0,0 +1,7 @@
+class AddOmniauthToUsers < ActiveRecord::Migration[8.0]
```

RuboCop (1.81.7)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)